controllers/update_pods.go

```diff
@@ -300,6 +314,10 @@ func deletePodsForUpdates(ctx context.Context, r *FoundationDBClusterReconciler,
 		}
 	}
 
+	// TODO(manfontan): if deletions size is larger than the fault tolerance of the cluster. The number of zones from
```

Comment:
This is unrelated to maxZonesWithUnavailablePods. I will work on this in a new PR. If that makes sense

Reply:
🤔 actually if we are using the default mode the number of zones will be one. So a check here won't add any additional security.